patches.json

```diff
@@ -238,6 +238,9 @@
     },
     "Fix for multi-site configuration issue": {
       "2.2.4": "MAGETWO-92926__fix_for_multi-site_configuration_issue__2.2.4.patch"
+    },
+    "Fix PayPal issue with region": {
+      "2.3.4": "MC-31387__fix_paypal_issue_with_region__2.3.4.patch"
```

Comment:
Will be this fix present in 2.3.4 patch version?
If not you need to change constraint to ">=2.3.4 < 2.3.5"

Reply:
@oshmyheliuk yes, fix will be present in patch versions starting from 2.3.4-p1

Reply:
@oshmyheliuk, the fix will be ported into the 2.3.4 security patch. 
cc: @RyanZolper, @smiverma